config/esp32/components/chip/Kconfig

```diff
@@ -620,6 +620,13 @@ menu "CHIP Device Layer"
             bool "Enable Rotating Device Identifier Support"
             default n
 
+        config ENABLE_ESP32_COMMISSIONABLE_DATA_PROVIDER
+            bool "Use ESP32 Commissionable Data Provider"
+            default n
+            help
+                Use commissionable data provider implementation for ESP32.
+                Default is test-mode CommissionableDataProvider in GenericConfigurationManagerImpl.
```

Comment:
You should encourage your customers to provide their own version. The "test-mode CommissionableDataProvider" is meant to never be used by real products since it relies on build-time values

Reply:
Sure, will do. Test-mode CommissionableDataProvider is kept just to not break the current commissioning flow.